app/src/main/java/org/p2p/wallet/solend/ui/earn/SolendEarnPresenter.kt

```diff
@@ -77,10 +78,14 @@ class SolendEarnPresenter(
     }
 
     override fun onDepositTokenClicked(deposit: SolendDepositToken) {
-        view?.showDepositTopUp(deposit)
+        if (!blockedErrorState) {
+            view?.showDepositTopUp(deposit)
+        }
     }
 
     private fun showDepositsWidgetError() {
+        timerJob?.cancel()
+        blockedErrorState = true
```

Comment:
blockedErrorState  -> widgetErrorState

Reply:
no it is block because we block clicks =)